packages/subscription-controller/src/types.ts

```diff
@@ -80,10 +80,40 @@ export type AuthUtils = {
   getAccessToken: () => Promise<string>;
 };
 
+export type PricingResponse = {
+  products: {
```

Comment:
we would need the types for individual items in this as mentioned here to be used in frotend. https://github.com/consensys-vertical-apps/va-mmcx-subscription-api/blob/main/src/subscriptions/dto/pricingDto.ts

Pls type out the same way